CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.94.0] - 2022-06-22
+
+### Added
+- Metric / dimension SQL validations via DW engine (@QMalcolm)
+- Support for PostgreSQL (@rexledesma)
+- Support for DuckDB (@plypaul)
+
+### Updated
+- CLI tests without mocks (@WilliamDee)
+- Backwards incompatible CLI configuration file changes for BigQuery (@WilliamDee)
```

Comment:
Oh hah I totally missed this. See above - it actually adds functionality so might be nice to call that out.

I kind of like a separate section for breaking changes when they're present. Seems nice.